### PR TITLE
fix wrappedObject.Delete panic

### PR DESCRIPTION
**Pull request by electricface**
https://github.com/gopherjs/vecty/issues/244

### CODE REVIEW

**Comment by flyn-org** [comment]
I ran into the same problem, although in my case the symptom ("property delete is not a function, ...") was present consistently on Epiphany (GNOME Web), Firefox, and Chromium. The pull request here fixed the problem for me.

File: dom_wasmjs_gopherjs.go [hunk 1]
```diff
@@ -26,6 +26,10 @@ var (
 	undefined = wrappedObject{js.Undefined()}
 )
 
+func init() {
+	js.Global().Call("eval", "$vectyDelete = function(obj, key) { delete(obj[key])}")
```
Comment (slimsag) [inline dom_wasmjs_gopherjs.go]: Sorry for the delayed review. I did review this a while back but waited to see what the resolution was for https://github.com/golang/go/issues/33079 and failed to post that over here.

Go 1.14 has added an explicit `Delete` operator that does not require using `eval` like in this PR: (see https://github.com/golang/go/issues/33079) and using `eval` has negative consequences, like needing a certain CSP (see https://github.com/golang/go/issues/26748)

I think the best option here is to use the new `Delete` method and require Go tip until Go 1.14 is released: https://tip.golang.org/pkg/syscall/js/#Value.Delete

If anyone can update this PR to do that, I can merge this.

Comment (slimsag) [reply dom_wasmjs_gopherjs.go]: Sent https://github.com/gopherjs/vecty/pull/256 to fix this.

**Comment by slimsag** [comment]
https://github.com/gopherjs/vecty/pull/256 supersedes this PR, thank you for taking the time to make a contribution though!

**Pull request closed without merging**